packages/serverpod/lib/src/database/database.dart

```diff
@@ -254,6 +266,7 @@ class Database {
   ///
   /// You are responsible to sanitize the query to avoid SQL injection. Always
   /// use QueryParameters for passing values to SQL queries.
+  @internal
```

Comment:
I think the `unsafe*` ones should stay exposed, as there is no alternative in the repositories.

The others all have a nicely typed alternative which should be used instead.